internal/traffic/clients.go

```diff
@@ -0,0 +1,16 @@
+package traffic
+
+import (
+	"context"
+	"io"
+)
+
+// ClientLoadBalancer intercepts requests for connected client.
+type ClientLoadBalancer[C ClientConnection] interface {
```

Comment:
This still need to be implemented. 

This is essentially the `map[string]*websocket.Conn` but more abstract.